cmd/kubeadm/app/phases/etcd/spec/zz_generated.deepcopy.go

```diff
@@ -1,307 +0,0 @@
-// +build !ignore_autogenerated
```

Comment:
Was this intentional?

Reply:
No, fixing now. Not sure why this was generated.

Reply:
https://github.com/kubernetes/kubernetes/issues/55500 for this file not being regenerated